src/app/docs/input-box/input-box-docs.component.html

```diff
@@ -0,0 +1,222 @@
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo
+    [skyTheme]="modernLightTheme"
+  >
+    <sky-fluid-grid>
+      <sky-row>
+        <sky-column
+          screenSmall="6"
+        >
+          <sky-input-box>
+            <label
+              class="sky-control-label"
+              for="myField"
+            >
+              Input label
+            </label>
+            <input
+              class="sky-form-control"
+              id="myField"
+              type="text"
+            >
+          </sky-input-box>
+        </sky-column>
+      </sky-row>
+    </sky-fluid-grid>
+  </sky-docs-demo>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use input box when users need to enter data in forms. The Input box component provides styling for many types of data entry, from basic single-line text, to text areas, to other SKY UX inputs like <a href="https://developer.blackbaud.com/skyux-datetime/docs/datepicker">Datepicker</a>.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use input box for data entry on forms."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/input-box/input-box-usage-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/input-box/input-box-anatomy.png"
+      >
+        <sky-docs-anatomy-item>Input container</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Input label</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Text value</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior and states"
+    >
+      <table class="sky-docs-table">
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Base
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-base.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Hover
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-hover.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Focus
```

Comment:
It looks like focus has a drop shadow?

Reply:
In the demo they definitely look the same; this branch needs to be updated from the master branch to pick up the new state styles.

Reply:
Just updated the branch.

Reply:
How do I see the difference in the demo? When I tab to the demo field, it looks the same as when I click the field in demo. both ways, the border is bolder and there is a drop shadow. Taking a second look at these images, it looks like the focus has a drop shadow and click does not. Does that mean the when we updated to get state styles from master that the demo wasn't updated properly?

Reply:
Following up on this because after looking at this again, I don't see a difference between focus and click in the demo. When I tab to the demo field, I get something that looks like our image for focus in the table. And when I click the demo field, I get something that looks the same. Clicking the field does NOT lose the drop shadow in the focus screenshot as the table suggests. Am I overlooking some way to differentiate between focus and click in the demo? Assuming not, is the behavior of the demo correct or is the table correct? @Blackbaud-AdamFunderburk, my best guess here is that the demo is correct and that we should combine the Focus and Click rows in the table into a single Hover/Click row. (Either that or use the Focus image in the click row too, but that seems unnecessary.) ... Please just let me know if I'm not following something here, but I *think* we need to tweak the table to be consistent.

Reply:
One more follow up... I noticed Paul's comment that `One general bit of feedback regarding the "click" state: I think this state may be better described as "active" rather than (or in addition to) "click." The reason I suggest "active" is because it aligns with the :active pseudo-class in CSS.` Do we need to change "Click" to "Active" in the Behavior and states table? (My previous question about Focus vs. Click/Active is still an open question even if we make this change.)

Reply:
Talked through this with Alex and he cleared things up. We decided to update "Click" to "Active" as per Paul's recommendation.